src/avkit.cs

```diff
@@ -16,86 +16,12 @@
 #else
 using AppKit;
 #endif
+#if MONOMAC
+using UIImage=AppKit.NSImage;
+#endif
 
 namespace AVKit {
 #if !MONOMAC
-	[NoTV]
```

Comment:
[re]move the `#if !MONOMAC` directive
this will reduce the size of the diff to review and makes it much easier to see any other changes